frontend/src/views/v-authorship.vue

```diff
@@ -422,6 +446,12 @@ export default {
         out.wasCodeLoaded = lineCnt <= COLLAPSED_VIEW_LINE_COUNT_THRESHOLD;
         out.fileType = file.fileType;
 
+        if (file.isIgnored) {
+          out.isIgnored = true;
+        } else {
+          out.isIgnored = false;
+        }
+
```

Comment:
`out.isIgnored = file.isIgnored` ?

Reply:
file.isIgnored and file.isBinary can take values `true | undefined`

`out.isIgnored = !!file.isIgnored` should do the trick!

Reply:
Seems good